rollup.config.js

```diff
@@ -66,6 +67,7 @@ export default [
         transforms: {
           dangerousForOf: true,
         },
+        objectAssign: 'Object.assign',
```

Comment:
[`Object.assign`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Object/assign) does not work on IE.